### ISSUE TITLE
Enter Ctrl + Z cause server crash

### ISSUE THREAD

**Issue by enip2473** [NTU-SP/SP-Hw1-release#4]
<img width="335" alt="image" src="https://user-images.githubusercontent.com/84924924/193233077-f1fa0e21-b301-4db4-996d-e8e0388e4fe4.png">

I think it is because this line does not handle case Ctrl + Z but only Ctrl + C. Do we have to handle it? This is not marked as TODO, but HackMD says that we have to handle invalid input. I wonder if this issue is included.

<img width="391" alt="image" src="https://user-images.githubusercontent.com/84924924/193233299-43ffd2f2-66f2-47a7-b7d8-04aedf8eac03.png">

**Comment by imingasco** [comment]
No, you don't need to handle cases like Ctrl+Z, Ctrl+D, ..., etc.
